testing/tests/DevExpress.localization/localization.base.tests.js

```diff
@@ -653,8 +654,13 @@ QUnit.test("input attr 'type' was not localized (Q588810)", function(assert) {
 
 QUnit.module("Localization number");
 
+QUnit.test("parse different positive and negative parts", function(assert) {
+    assert.equal(numberLocalization.parse("(10)", "#0;(#0)"), -10);
+    assert.equal(numberLocalization.parse("-10"), -10);
+    assert.equal(numberLocalization.parse("-10", "#0;(#0)"), -10);
+});
+
 QUnit.test("format: base", function(assert) {
-    //assert.equal(numberLocalization.format(1.2), "1.2");
```

Comment:
Why is this test case removed?

Reply:
I have just remove commented strings from the code